e2e/tests/functions.bats

```diff
@@ -36,7 +36,7 @@ load variables
     echo_to_log
     assert_success
 
-    run_with_retry "dispatch get runs node-hello-no-schema --json | jq '. | length'" 0 1 0
+    run_with_retry "dispatch get runs node-hello-no-schema --json | jq '. | length'" 0 5 5
```

Comment:
Deleting function is also an async operation. In case of docker driver, the delete takes a bit longer, and runs are only cleaned after it is deleted, so it may happen that one attempt is not enough.